app/components/rdf-input-fields/concept-selector.js

```diff
@@ -2,48 +2,16 @@ import InputFieldComponent from '@lblod/ember-submission-form-fields/components/
 
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
-import { tracked, cached } from '@glimmer/tracking';
+import { tracked } from '@glimmer/tracking';
 import { service } from '@ember/service';
 
 import { NamedNode, Literal } from 'rdflib';
 import { hasValidFieldOptions } from '@lblod/ember-submission-form-fields/utils/has-valid-field-options';
 import { task, timeout } from 'ember-concurrency';
-import { trackedFunction } from 'reactiveweb/function';
-import { use } from 'ember-resources';
 
 import { SEARCH_TIMEOUT } from 'frontend-lmb/utils/constants';
 import { FIELD_OPTION } from 'frontend-lmb/rdf/namespaces';
 import { showWarningToast } from 'frontend-lmb/utils/toasts';
-
-function getOptions() {
-  return trackedFunction(async () => {
```

Comment:
no matter what i tried, i couldn't entangle this.searchData with the tracked function. It never updated when its value changed, so i just rewrote the tracked function out of it and actively ask for it. Imo this also makes it more clear.